cirq/google/engine/engine.py

```diff
@@ -155,13 +155,18 @@ class Engine:
         set_program_labels
     """
 
-    def __init__(self,
-                 project_id: str,
-                 version: Optional[str] = None,
-                 discovery_url: Optional[str] = None,
-                 default_gcs_prefix: Optional[str] = None,
-                 proto_version: ProtoVersion = ProtoVersion.V1,
-                 service_args: Optional[Dict] = None) -> None:
+    def __init__(
+            self,
+            project_id: str,
+            version: Optional[str] = None,
+            discovery_url: Optional[str] = None,
+            default_gcs_prefix: Optional[str] = None,
+            proto_version: ProtoVersion = ProtoVersion.V1,
+            service_args: Optional[Dict] = None,
+            priority: int = 50,
+            processor_ids: Sequence[str] = ('xmonsim',),
```

Comment:
You need to document these additional parameters in the docstring. Explain that they are default values used if more specific ones are not specified on an individual call.